api/src/test/java/org/openmrs/module/bedmanagement/BedManagementServiceIntegrationTest.java

```diff
@@ -35,8 +35,8 @@ public void beforeAllTests() throws Exception {
 
     @Test
     public void getAllLocationsBy_gets_locations_for_a_tag() {
-        List<AdmissionLocation> admissionLocationList = bedManagementService.getAllAdmissionLocations();
-        assertThat(admissionLocationList.size(), is(2));
+        List<AdmissionLocation> admissionLocationList = bedManagementService.getAdmissionLocations();
+        assertThat(admissionLocationList.size(), is(3));
```

Comment:
The previous implementation has a limitation that it will not return admission location if it does not have any bed associated with the admission location. Actually, in test data set we have "Cardio ward on first floor", "Cardio ward on third floor" and "Orthopaedic ward". In which "Cardio ward on third floor" has totalBeds=0 that not return by the previous implementation.